public/components/Flyout/flyouts/components/AlertsDashboardFlyoutComponent.js

```diff
@@ -45,14 +46,18 @@ import { queryColumns } from '../../../../pages/Dashboard/utils/tableUtils';
 import { DEFAULT_PAGE_SIZE_OPTIONS } from '../../../../pages/Monitors/containers/Monitors/utils/constants';
 import queryString from 'query-string';
 import { MAX_ALERT_COUNT } from '../../../../pages/Dashboard/utils/constants';
+import { SEVERITY_OPTIONS } from '../../../../pages/CreateTrigger/utils/constants';
+import { TABLE_TAB_IDS } from '../../../../pages/Dashboard/components/FindingsDashboard/utils';
+import FindingsDashboard from '../../../../pages/Dashboard/containers/FindingsDashboard';
 
 export const DEFAULT_NUM_FLYOUT_ROWS = 10;
 
 export default class AlertsDashboardFlyoutComponent extends Component {
   constructor(props) {
     super(props);
-    const { location, monitor_id } = this.props;
-
+    const { location, monitors, monitor_id } = this.props;
+    const monitor = _.get(_.find(monitors, { _id: monitor_id }), '_source');
+    const monitorType = _.get(monitor, 'monitor_type', MONITOR_TYPE.QUERY_LEVEL);
```

Comment:
Why is this only for `QUERY_LEVEL`?

Reply:
On `line 60`, right? The 3rd parameter in the `_.get()` method is a default value to return if `monitor.monitor_type` is `undefined`.

Reply:
Ah ok thanks, I had misunderstood that